torch/csrc/api/include/torch/nn/options/activation.h

```diff
@@ -16,5 +16,15 @@ struct ELUOptions {
   TORCH_ARG(bool, inplace) = false;
 };
 
+// ============================================================================
+
+/// Options for Hardshrink functional and module.
+struct HardshrinkOptions {
+  HardshrinkOptions() {}
+
+  // the lambda value for the Hardshrink formulation. Default: 0.5
```

Comment:
nit: we need `///` instead of `//` in order for the comment to show up in the docs